site/config/course_template.json

```diff
@@ -23,6 +23,7 @@
         "room_seating_gradeable_id": "",
         "auto_rainbow_grades": false,
         "queue_enabled": false,
-        "queue_contact_info": false
+        "queue_contact_info": false,
+        "queue_message": "Welcome to the office hours/lab queue. Choose a queue name and enter the code secret code you have been provided."
```

Comment:
Do we want to have this default message for all Office Hours queues or do you want this to just be a power user feature?

Reply:
Now that I think about it I agree. This message adds nothing so I will make it a power user feature